specification/cognitiveservices/data-plane/Speech/VideoTranslation/preview/2024-05-20-preview/examples/list_iterations.json

```diff
@@ -0,0 +1,38 @@
+{
+    "parameters": {
+      "Endpoint": "https://eastus.api.cognitive.microsoft.com/",
+      "Ocp-Apim-Subscription-Key": "{API Key}",
+      "api-version": "2024-05-20-preview"
+    },
+    "responses": {
+      "200": {
+        "headers": {},
+        "body": {
+          "value": [
+            {
+              "id": "Initial",
+              "description": "Translate video from en-US to zh-CN, fix segment issue.",
+              "status": "Succeeded",
+              "createdDateTime": "2023-04-10T05:30:00.000Z",
+              "lastActionDateTime": "2023-04-10T05:15:30.000Z"
+            },
```

Comment:
Do you mean list API will return less information than get API?

Reply:
Yes, for iterations, list will not response result.